airmozilla/manage/templates/manage/survey_edit_questions.html

```diff
@@ -0,0 +1,82 @@
+{% extends 'manage/surveys.html' %}
+
+{% block manage_title %}
+    Questions for "{{survey.name}}"
+{% endblock %}
+
+{% block site_css %}
+  {{ super() }}
+  {% compress css %}
+  <link href="{{ static('select2/select2.css') }}" rel="stylesheet" type="text/css">
+  <link href="{{ static('select2/select2-bootstrap.css') }}" rel="stylesheet" type="text/css">
+  <link href="{{ static('manage/css/survey.css') }}" rel="stylesheet" type="text/css">
+  {% endcompress %}
+{% endblock %}
+
+{% block site_js %}
+  {{ super() }}
+  <script src="{{ static('select2/select2.min.js') }}"></script>
+  {% compress js %}
+    <script src="{{ static('manage/js/survey-edit.js') }}"></script>
+  {% endcompress %}
+{% endblock %}
+
+{% block content_class %}col-md-6{% endblock %}
```

Comment:
If you remove this line, I think you get a full width page and then the textarea boxes get a bit more breathing room.